### PR TITLE
Add Job docker getter

### PR DESCRIPTION
**Pull request by orlandohohmeier**
/cc @jfurrow

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @Poltergeist

File: src/js/structs/__tests__/Job-test.js [hunk 1]
```diff
@@ -35,13 +35,37 @@ describe('Job', function () {
...
 
+  describe('#getDocker', function () {
+
+    it('returns the docker configuration', function () {
+      let job = new Job({id: 'foo', run: {docker: {image: 'busybox'}}});
+
+      expect(job.getDocker()).toEqual({image: 'busybox'});
+    });
+
+    it('defaults to an empty object if property is undefined', function () {
```
Comment (pierlo-upitup) [inline src/js/structs/__tests__/Job-test.js]: why is an empty object better than `null` ?

Comment (orlandohohmeier) [inline src/js/structs/__tests__/Job-test.js]: It reduces the number of checks needed when accessing properties (e.g. `job.getDocker().image`)

**Comment by orlandohohmeier** [comment]
**mesosphere-ci** retest this please

**Comment by orlandohohmeier** [comment]
**mesosphere-ci** retest this please

**Pull request merged**